app/assets/javascripts/media_player_wrapper/mejs4_plugin_playlist_items.es6

```diff
@@ -79,6 +80,27 @@ Object.assign(MediaElementPlayer.prototype, {
    * @type {Object}
    */
   playlistItemsObj: {
+    /**
+     * Add click listeners forplaylist item links
+     * @function addRelatedItemListeners
+     * @return {void}
+     */
+    addRelatedItemListeners() {
+      // Handle click on entire RelatedItems area
+      // Filter only <a> element clicks; disregard all others
+      if (this.$relatedItems) {
+        this.$relatedItems.on('click', (e) => {
+          if (e.target.nodeName === 'A') {
+            // find correct playlist item in sidebar and click it.
+            let playlist_id  = e.target.dataset.playlistId
```

Comment:
Not to get all nitpicky, but in general for our JavaScript can we follow camelCase convention for variables?

Reply:
Would eslint or prettier pick up on this?

Reply:
@cjcolvar I don't think either will change variable titles.  No big deal for now, just made the comment as a 'in the future' type thing.   Let's get this sucker working for now.